contracts/src/Teleporter/upgrades/tests/TeleporterUpgradeableTests.t.sol

```diff
@@ -159,6 +121,33 @@ contract TeleporterUpgradeableTest is TeleporterRegistryTest {
         assertEq(app.getMinTeleporterVersion(), latestVersion + 1);
     }
 
+    function _updateMinTeleporterVersionSuccess(
+        uint256 newMinTeleporterVersion
+    ) internal virtual {
+        vm.expectEmit(true, true, true, true, address(app));
+        emit MinTeleporterVersionUpdated(
+            app.getMinTeleporterVersion(),
+            newMinTeleporterVersion
+        );
+        app.updateMinTeleporterVersion(newMinTeleporterVersion);
+    }
+
+    function _pauseTeleporterAddressSuccess(
+        address teleporterAddress_
+    ) internal virtual {
+        vm.expectEmit(true, true, true, true, address(app));
+        emit TeleporterAddressPaused(teleporterAddress_);
+        app.pauseTeleporterAddress(teleporterAddress_);
+    }
+
+    function _unpauseTeleporterAddressSuccess(
+        address teleporterAddress_
+    ) internal virtual {
+        vm.expectEmit(true, true, true, true, address(app));
+        emit TeleporterAddressUnpaused(teleporterAddress_);
+        app.unpauseTeleporterAddress(teleporterAddress_);
+    }
```

Comment:
Can we consolidate these with the functions defined in contracts/src/Teleporter/upgrades/tests/TeleporterOwnerUpgradeableTests.t.sol?

Reply:
passing in the app and consolidated in `TeleporterUpgradeableTests`